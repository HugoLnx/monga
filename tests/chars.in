int main() {
	char a;
	a = 'a';
	
	while (a > 0) {
		if (a >= '!') {
			printf("char: %c\n", a);
		}
		else {
			printf("int: %d\n", a);
		}
		a = a - 1;
	}
}