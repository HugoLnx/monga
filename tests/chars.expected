char: a
char: `
char: _
char: ^
char: ]
char: \
char: [
char: Z
char: Y
char: X
char: W
char: V
char: U
char: T
char: S
char: R
char: Q
char: P
char: O
char: N
char: M
char: L
char: K
char: J
char: I
char: H
char: G
char: F
char: E
char: D
char: C
char: B
char: A
char: @
char: ?
char: >
char: =
char: <
char: ;
char: :
char: 9
char: 8
char: 7
char: 6
char: 5
char: 4
char: 3
char: 2
char: 1
char: 0
char: /
char: .
char: -
char: ,
char: +
char: *
char: )
char: (
char: '
char: &
char: %
char: $
char: #
char: "
char: !
int: 32
int: 31
int: 30
int: 29
int: 28
int: 27
int: 26
int: 25
int: 24
int: 23
int: 22
int: 21
int: 20
int: 19
int: 18
int: 17
int: 16
int: 15
int: 14
int: 13
int: 12
int: 11
int: 10
int: 9
int: 8
int: 7
int: 6
int: 5
int: 4
int: 3
int: 2
int: 1